diesel/src/mysql/connection/url.rs

```diff
@@ -29,8 +29,10 @@ impl ConnectionOptions {
             return Err(connection_url_error());
         }
 
-        let host = match url.host_str() {
-            Some(host) => Some(try!(CString::new(host.as_bytes()))),
+        let host = match url.host() {
+            // Avoid brackets around IPv6 addresses.
```

Comment:
Can we delete this comment? The commit message should have all the context needed.